crates/pallet-feeds/src/lib.rs

```diff
@@ -51,20 +56,22 @@ mod pallet {
     pub(super) type ObjectMetadata = Vec<u8>;
 
     /// Total amount of data and number of objects stored in a feed
-    #[derive(Decode, Encode, TypeInfo, Default)]
+    #[derive(Decode, Encode, TypeInfo, Default, PartialEq, Eq)]
     #[cfg_attr(feature = "std", derive(Debug))]
-    pub(super) struct TotalObjectsAndSize {
+    pub struct TotalObjectsAndSize {
```

Comment:
Why does it need to be `pub` if other fields in it are `pub(super)` anyway?

Reply:
should whole struct be just `pub`? I removed `super` because I could not import struct in `tests.rs`, this way it works

Reply:
Ah, if you want to access things then yes. See, this is why I suggest to move it to integration tests, so your test cases have no access to things they shouldn't have access to and you can verify that you can in fact read the fields you intended to read using just public API and nothing else.